### ISSUE TITLE
sieve incorrectly reports "header 'resent-from': not a valid header for an address test"

### ISSUE THREAD

**Issue by gdmalet** [cyrusimap/cyrus-imapd#3307]
Branch cyrus-imapd-3.2 (a7aaf65, currently v3.2.5), but not the current master (06f5d28) nor v3.0 nor v2.5, has this error in it (missing comma). This was introduced back in May 2018 by 31bb54a1 -- I'm amazed this still exists! I noticed it by updating our v3.2 installation recently to v3.2.5 and our logs now contain many examples of the bogus error.

Also, running sievec on something containing this:
```[...]
if address :all :comparator "i;ascii-casemap" :is ["From", "Sender", "Resent-From"] "xxx@yyy.com"  { 
    keep;
    stop;
}
```
produces this:
```
Unable to parse script: script errors:
line 5: header 'resent-from': not a valid header for an address test
```

Here's a patch:

```
diff --git a/sieve/sieve.y b/sieve/sieve.y
index 9cda9bb..fa83010 100644
--- a/sieve/sieve.y
+++ b/sieve/sieve.y
@@ -2169,7 +2169,7 @@ static int verify_addrheader(sieve_script_t *sscript, char *hdr)
         "from", "sender", "reply-to",   /* RFC5322 originator fields */
         "to", "cc", "bcc",              /* RFC5322 destination fields */
         "message-id", "in-reply-to",    /* RFC5322 identification fields */
-        "references"
+        "references",
         "resent-from", "resent-sender", /* RFC5322 resent fields */
         "resent-to", "resent-cc", "resent-bcc",
         "return-path",                  /* RFC5322 trace fields */
```

Thanks,
g

**Comment by gdmalet** [comment]
This was fixed by 52718905c5 in master.

**Comment by elliefm** [comment]
Oh wow.  Thanks for tracking that down, I have cherry-picked @dilyanpalauzov's fix from master, and it will be included in 3.2.6

**Comment by dilyanpalauzov** [comment]
As there is nothing more to do here, this case can be closed.
